src/plugin.js

```diff
@@ -18,8 +18,8 @@ let plugin = (hook, vm) => {
         next(htmlElement.innerHTML);
     });
 
-    hook.doneEach(function () {
-        mermaid.init({}, '.mermaid');
+    hook.doneEach(() => {
+        mermaid.run(mermaidConf);
     });
```

Comment:
```js
// Invoked one time after rendering the initial page
    hook.ready(function () {
      // ...
    });
```

This should be enough to render all diagrams at once.

Reply:
You're right! I modified my commit.